src/WalkingModule/src/Module.cpp

```diff
@@ -442,6 +446,14 @@ bool WalkingModule::configure(yarp::os::ResourceFinder& rf)
     m_qDesired.resize(m_robotControlHelper->getActuatedDoFs());
     m_dqDesired.resize(m_robotControlHelper->getActuatedDoFs());
 
+    yarp::os::Bottle navigationOptions = rf.findGroup("NAVIGATION");
+    
+    // start the threads used for computing navigation needed infos
+    if(!m_navHelper.init(navigationOptions, m_leftInContact, m_rightInContact))
+    {
+        yError() << "[WalkingModule::configure] Could not initialize the Navigation Helper";
+    }
```

Comment:
I would call this only if ``navigationOptions`` is not null, thus allowing the user to skip this part if not needed.

Reply:
Yep. done thanks